include/seqan3/std/charconv

```diff
@@ -859,6 +859,23 @@ inline std::to_chars_result to_chars(char * first, char * last, value_type value
     return seqan3::detail::to_chars_itoa(first, last, value, is_signed<value_type>());
 }
 
+//!\brief std::to_chars overload for floating point via a stringstream.
+//!\ingroup charconv
+template <seqan3::FloatingPoint floating_point_type>
+inline std::to_chars_result to_chars(char * first, char * last, floating_point_type value, int = 10) noexcept
+{
+    std::ostringstream ss;
+    ss << value;
+    auto str = ss.str();
+
+    if (last - first < static_cast<int16_t>(str.size()))
```

Comment:
Why `int16_t` instead of `std::ptrdiff_t`?

Reply:
I used ptrdiff_t instead. Although I know that a proper arithmetic value will never be longer than 316 chars, this might avoid overflows.